lib/rucio/client/downloadclient.py

```diff
@@ -460,31 +464,11 @@ def _download_item(self, item, trace, log_prefix=''):
                     continue
                 end_time = time.time()
                 success = (status == 0)
-                if success and not item.get('ignore_checksum', False):
```

Comment:
does this disable checksumming for everything, or only for the archive extraction part?

Reply:
only the zip download checksum validation